data/mods/More_Makeshift_Items/mb_item_groups.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "type" : "item_group",
+    "id" : "pawn",
+    "items":[
+      ["sword_bayonet", 10],

```

Comment:
Have you tested if this works? I'm suspicious this will replace all the other items?